app/app.go

```diff
@@ -1062,3 +1112,29 @@ func initParamsKeeper(appCodec codec.BinaryCodec, legacyAmino *codec.LegacyAmino
 func (app *StrideApp) SimulationManager() *module.SimulationManager {
 	return app.sm
 }
+
+// ConsumerApp interface implementations for e2e tests
+// GetConsumerKeeper implements the ConsumerApp interface.
+func (app *StrideApp) GetConsumerKeeper() ccvconsumerkeeper.Keeper {
+	return app.ConsumerKeeper
+}
+
+// // GetE2eBankKeeper implements the ConsumerApp interface.
```

Comment:
nit: rm unless these are needed

Reply:
It would be fine to remove - this code is for unit test and if it's not in unit test, it can be removed.

Reply:
Ok